manifests/p/Pulumi/Pulumi/3.26.1/Pulumi.Pulumi.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: Pulumi.Pulumi
+PackageVersion: 3.26.1
+PackageLocale: en-US
+Publisher: Pulumi Corp
+PackageName: Pulumi
+PackageUrl: https://www.pulumi.com
+License: Apache License 2.0
+LicenseUrl: https://github.com/pulumi/pulumi/blob/master/LICENSE
+ShortDescription: Pulumi CLI for managing modern infrastructure as code
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/pulumi/pulumi-winget/releases/download/v3.26.1/pulumi-3.26.1-windows-x64.msi
+  InstallerSha256: 72F32545E13E7D8ED97682ABEE4331BAEDEDC2826ABBDCB8F4CF207CE2875178

```

Comment:
```suggestion
  InstallerSha256: 72F32545E13E7D8ED97682ABEE4331BAEDEDC2826ABBDCB8F4CF207CE2875178
  ProductCode: "{D3D9E7D4-2FB8-417C-BA3A-588E47FF5C45}"
```

Reply:
@jedieaston I don't have access to the @pulumi-bot directly so I can't accept this suggestion. Can you please apply it if possible? otherwise I would have to close and re-open with a new manifest

Reply:
We can't accept suggestions on the behalf of contributors, unfortunately :(